### PR TITLE
Added yarn to compile build.

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: compile-build [hunk 1]
```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-set -e
+set -e # Exit immediately if something fails
+export NODE_ENV=production
+export RAILS_ENV=development
```
Comment (emrojo) [inline compile-build:5]: Why is there this difference between NODE_ENV and RAILS_ENV? This is going to be used to compile a production release

File: compile-build [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+echo 'Updating yarn'
+bundle exec yarn
```
Comment (sdjmchattie) [inline compile-build:7]: Why does this need `bundle exec`?  I don't think `yarn` is a Bundler dependency.

**Review by emrojo — COMMENTED** [review]
One comment, but because I probably don't understand what was the change for.

File: compile-build (hunk 1 above)
Comment (stevieing) [reply compile-build:5]: I have copied these from Sequencescape.

File: compile-build (hunk 2 above)
Comment (stevieing) [reply compile-build:7]: Yarn is needed to build the node dependencies. So bundle exec yarn will do this.

File: compile-build (hunk 1 above)
Comment (stevieing) [reply compile-build:5]: So it might not be needed ... The RAILS_ENV is not the same as deploying. This is generating the assets not the environments.

**Pull request merged**